generated/proto/encodingpb/int.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+option go_package = "github.com/xichen2020/eventdb/generated/proto/encodingpb";
+
+import "github.com/xichen2020/eventdb/generated/proto/encodingpb/types.proto";
+
+package encodingpb;
+
+message IntMeta {
+    EncodingType encoding = 1;
+    int64 num_records = 2;
+    // Bytes per dictionary value since we can't do bit packing 
+    // in the int dictionary yet. See TODO below.
+    int64 bytes_per_dictionary_value = 3;
```

Comment:
Probably better to leave as is since there are > 1 fields required (e.g. `bytes_per_dictionary_value` and `min_value` for dict encoding). Can't use `oneof` w/o further grouping the fields into their own messages.

Reply:
Yeah I meant grouping the fields needed for dictionary encoding and those for delta encoding as different message types, and use `oneof` for them. Take it or leave it though.